gammapy/maps/hpxmap.py

```diff
@@ -231,6 +263,6 @@ def make_hdu(self, **kwargs):
                                     array=np.arange(data.shape[-1])))
```

Comment:
Should there be an else clause here? Or is the "do nothing" appropriate here?
A few lines above in `self.geom._ipix`, you're accessing a private data member `_ipix`. Can that be avoided.

Reply:
Yes the absence of an else clause is appropriate here.  The reason being that the logic for making the HDU is split between this method and the `_make_cols` helper method.

Access to `_ipix` could be avoided but probably requires some more careful thought toward how to expose this information in a public method.  I'd propose we defer that for now.